src/lib/ibcclient.ts

```diff
@@ -622,6 +649,7 @@ export class IbcClient {
     clientId: string,
     remoteClientId: string
   ): Promise<CreateConnectionResult> {
+    this.logger.info(`Connection open init: ${clientId} => ${remoteClientId}`);
```

Comment:
I like this format. Let's use a similar one (myId => remoteId) for all the connection handshake logs (and channel handshake too)